jupyter_sphinx/ast.py

```diff
@@ -392,9 +392,11 @@ def html(self):
         #       parsing script tags, and not just grabbing their innerHTML
         # https://github.com/jupyter-widgets/ipywidgets/blob/master/packages/html-manager/src/libembed.ts#L36
         return ipywidgets.embed.snippet_template.format(
-            load="", widget_views="", json_data=json.dumps(self["state"])
+            load="", 
+            widget_views="", 
+            json_data=json.dumps(self["state"]).replace("</script>", "&lt/script>")
```

Comment:
Would this perhaps be more readable?
```suggestion
            json_data=json.dumps(self["state"]).replace("</script>", r"<\/script>")
```